cmd/kubeadm/app/master/manifests.go

```diff
@@ -304,7 +304,7 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	}
 
 	command = append(getComponentBaseCommand(apiServer),
-		"--insecure-bind-address=127.0.0.1",
+		"--insecure-port=0",
```

Comment:
Is it possible to get the "randomly selects a port" behavior? It would be useful for testing.